src/components/PrintModal/PrintModal.js

```diff
@@ -446,40 +485,21 @@ class PrintModal extends React.PureComponent {
                 label={t('option.print.includeComments')}
               />
             </form>
+
           </div>
-          <div className="total">
-            {isPrinting ? (
-              <div>{`${t('message.processing')} ${count}/${
-                pagesToPrint.length
-              }`}</div>
-            ) : (
-              <div>
-                {t('message.printTotalPageCount', {
-                  count: pagesToPrint.length,
-                })}
-              </div>
-            )}
-          </div>
+          {this.state.allowWatermarkModal && <button data-element="applyWatermark" className="apply-watermark" onClick={() => this.setWatermarkModalVisibility(true)}>{t('option.print.addWatermarkSettings')}</button> }
           <div className="buttons">
-            <div
-              className="button"
-              onClick={this.createPagesAndPrint}
-              disabled={count > -1}
-            >
-              {t('action.print')}
+            <div className="total">
+              {isPrinting
+                ? <div>{`${t('message.processing')} ${count}/${pagesToPrint.length}`}</div>
+                : <div>{t('message.printTotalPageCount', { count: pagesToPrint.length })}</div>
+              }
             </div>
-            {isPrinting ? (
-              <div className="button" onClick={this.cancelPrint}>
-                {t('action.cancel')}
-              </div>
-            ) : (
-              <div className="button" onClick={this.closePrintModal}>
-                {t('action.close')}
-              </div>
-            )}
+            <button className="button" onClick={event => this.createPagesAndPrint(event)} disabled={count > -1}>{t('action.print')}</button>
```

Comment:
Why do we need to make this an inline function, can we do `onClick={this.createPagesAndPrint}` instead?

Reply:
I don't think I may be on the same page as you on the definition of "inline function".

Here is what I see from Googling: 
![image](https://user-images.githubusercontent.com/54084743/64722594-a4ab3a00-d483-11e9-8322-9c810be611e9.png)
https://stackoverflow.com/questions/19159703/what-is-the-difference-between-anonymous-and-inline-functions-in-javascript

I believe you are asking why I'm choosing to use an arrow function over not using an arrow function and doing this instead: {this.createPagesAndPrint} ?

By doing {this.createPagesAndPrint}, I'm seeing that "this" is undefined inside that function when it is called.
![image](https://user-images.githubusercontent.com/54084743/64722884-2e5b0780-d484-11e9-9cdb-0ee892c88644.png)

Reply:
as discussed. i have made majority of the code i 've written to match what is in the code base.